### PR TITLE
chore: show topup/increase option ony if available

### PR DESCRIPTION
**Pull request by bumi**
The backends without channel management do not have support for this.

+ could those boxes be reusable components?
+ could the channel pages check if channel management is available and if not redirect to home?

### CODE REVIEW

File: frontend/src/screens/wallet/Receive.tsx [hunk 1]
```diff
@@ -238,11 +240,13 @@ export default function Receive() {
               )}
             </div>
           </CardContent>
-          <CardFooter className="flex justify-end">
-            <Link to="/channels/incoming">
-              <Button variant="outline">Increase</Button>
-            </Link>
-          </CardFooter>
+          {hasChannelManagement && (
```
Comment (rolznz) [inline frontend/src/screens/wallet/Receive.tsx:243]: Thanks, I didn't think about this :scream: 

But we should hide the entire card, right? "Receiving capacity" makes no sense for LNClients without channel management

Comment (bumi) [reply frontend/src/screens/wallet/Receive.tsx:243]: true, stupid me... :) 😱

**Comment by rolznz** [comment]
> could those boxes be reusable components?

I don't think the design is finalized yet, so it might be best to wait for that

> could the channel pages check if channel management is available and if not redirect to home?

We could do this, but there should be no way to access those pages normally (as long as we don't forget to implement these checks :sweat_smile: )

File: frontend/src/screens/wallet/index.tsx [hunk 2]
```diff
@@ -120,11 +122,13 @@ function Wallet() {
               )}
             </div>
           </CardContent>
-          <CardFooter className="flex justify-end">
-            <Link to="/channels/incoming">
-              <Button variant="outline">Increase</Button>
-            </Link>
-          </CardFooter>
+          {hasChannelManagement && (
```
Comment (rolznz) [inline frontend/src/screens/wallet/index.tsx:125]: I think this one still needs to be updated

**Review by rolznz — APPROVED** [review]
tACK

**Pull request merged**